x-pack/filebeat/module/threatintel/_meta/kibana/7/lens/06744e90-df52-11eb-8f2b-753caedf727d.json

```diff
@@ -47,7 +47,7 @@
             "filters": [],
             "query": {
                 "language": "kuery",
-                "query": "event.dataset:\"threatintel.recordedfuture\" and threatintel.indicator.type:ipv6-addr"
+                "query": "event.dataset:\"threatintel.recordedfuture\" and threat.indicator.type:ipv6-addr"
```

Comment:
This seems to still reference threatintel.recordedfuture, is that one correct?

Reply:
the dataset is still called `threatintel.recordedfuture`, unless we also change the module name. Is that something we want to do?

Reply:
All good here :)